lib/compiler/src/compile.erl

```diff
@@ -280,6 +280,8 @@ expand_opt(no_module_opt=O, Os) ->
     [O,no_recv_opt | Os];
 expand_opt({check_ssa,Tag}, Os) ->
     [check_ssa, Tag | Os];
+expand_opt(beam_docs, Os) ->
+    [beam_docs, Os];
```

Comment:
```suggestion
    [beam_docs | Os];
```

Reply:
actually I think you can delete this change all together

Reply:
you mean that we always add the compiled docs to the beam file? (not conditionally as it is now?)